internal/provider/resource_status_page.go

```diff
@@ -0,0 +1,280 @@
+package provider
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/squadcast/terraform-provider-squadcast/internal/api"
+	"github.com/squadcast/terraform-provider-squadcast/internal/tf"
+)
+
+func resourceStatusPage() *schema.Resource {
+	return &schema.Resource{
+		Description: "Status page resource.",
+
+		CreateContext: resourceStatusPageCreate,
+		ReadContext:   resourceStatusPageRead,
+		UpdateContext: resourceStatusPageUpdate,
+		DeleteContext: resourceStatusPageDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceStatusPageImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "Status page id.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"team_id": {
+				Description:  "Team id.",
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: tf.ValidateObjectID,
+			},
+			"name": {
+				Description:  "Status page name.",
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"description": {
+				Description: "Description.",
```

Comment:
Change the description to: "status page description". 
a bit more description for all will be good as it is going to be in our doc as well